table: Add support for aria-rowcount and aria-rowindex (#1424)
steelthreads/agds-next: .changeset/giant-baboons-appear.md

---
'@ag.ds-next/react': patch
---

> table: Add support for aria-rowcount and aria-rowindex

Reviewer: ```suggestion
table: Added support for `aria-rowcount` and `aria-rowindex` on the `Table` component
```
